packages/flutter_tools/templates/app_shared/ios.tmpl/Runner.xcodeproj/xcshareddata/xcschemes/Runner.xcscheme

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Scheme
-   LastUpgradeVersion = "1020"
+   LastUpgradeVersion = "1300"
```

Comment:
out of curiosity, what is 1020 and 1300 mean? are these constant codes?

Reply:
It's the version of Xcode that last checked if anything needed to be updated.  The last time we updated this was Xcode 10.2.  This change updates to Xcode 13.0.